### PR TITLE
Modulo-0 completado

### PR DESCRIPTION
**Pull request by AgustinaNahas**

### CODE REVIEW

File: 00-HTML-CSS-basics/css 2.1.1/css.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
+html { 
+	size: 14px;
```
Comment (nicolasepiscopo) [inline 00-HTML-CSS-basics/css 2.1.1/css.css:2]: You should try to unify the indentation, consider 4 spaces as standard for css, js and html. You can configure your favorite editor to transform tabs into "n spaces".

File: 00-HTML-CSS-basics/css 2.1.1/css.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+	font-family: Comic Sans MS;
```
Comment (nicolasepiscopo) [inline 00-HTML-CSS-basics/css 2.1.1/css.css]: Use fallback fonts in order to control side effects of multiple operative systems fonts. Check out this article: http://www.w3schools.com/cssref/css_websafe_fonts.asp

File: 00-HTML-CSS-basics/css 2.5/mock.css [hunk 3]
```diff
@@ -0,0 +1,227 @@
+html { 
+	font-family: Tahoma, Geneva, sans-serif;
+  color:white;
```
Comment (nicolasepiscopo) [inline 00-HTML-CSS-basics/css 2.5/mock.css]: be careful with readibility, fix indentations and after every colon ':' you should add a space.

File: 00-HTML-CSS-basics/css 2.5/mock.css [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+  }
+}
+
+/* Large screens (1024px) */
+@media (max-width: 2500px){ 
+  html { font-size: 100%; }
+}
+
+@media (min-width: 2500px) { 
+  html { font-size: 200%; }
```
Comment (nicolasepiscopo) [inline 00-HTML-CSS-basics/css 2.5/mock.css]: is a good practice not to write css code in the same line. You should break it into multiple conserving indentation lines like: 

```
html { 
    font-size: 200%; 
}
```

File: 00-HTML-CSS-basics/css 2.5/mock.css [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+    width: 95%;
+    display: inline-block;
+  }
+
+  #titulo {
+    width: 95%;
+    display: inline-block;
+  }
+
+  #texto {
```
Comment (nicolasepiscopo) [inline 00-HTML-CSS-basics/css 2.5/mock.css]: I've seen some definitions in english, some others in spanish, and others in a mix of them. You use english only words when you are writing code.

File: 00-HTML-CSS-basics/css 2.5/mock.css [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+      -o-transition: all 0.3s ease-in-out;
+        -ms-transition: all 0.3s ease-in-out;
+          transition: all 0.3s ease-in-out;
+}
+.descr {
+  color: #fff;
+  font-family: arial, helvetica, sans-serif;
+  font-weight: bold;
+  -webkit-transition: all 0.3s ease-in-out;
+    -moz-transition: all 0.3s ease-in-out;
```
Comment (nicolasepiscopo) [inline 00-HTML-CSS-basics/css 2.5/mock.css]: Take indentation very seriously, in a real project you should follow strict coding rules and standards, and one of them is a standarized indentation.

File: 00-HTML-CSS-basics/css 2.5/mock.html [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+			<div class="galeria">
+				<div class="boxDeImagen">
+					<a href="http://www.buenosaires.gob.ar/" target="_blank">
+						<span class="screen">
+						<p class="descr">
+							Avenida 9 de Julio
+						</p>
+						</span>
+						<img src="http://crazy-frankenstein.com/free-wallpapers-files/city/buenos-aires-wallpapers/buenos-aires-wallpapers-1024x768.jpg">
```
Comment (nicolasepiscopo) [inline 00-HTML-CSS-basics/css 2.5/mock.html]: You must always make use of alt property in img elements in order to improve the accessibility

File: 00-HTML-CSS-basics/html 1.1.4/html1.1.4.html [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+		<title>Expensas</title>
+		<meta charset="utf-8">
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+	</head>
+	<header><strong>Expensas</strong></header>
+	<body>
+		<div>
+			<table width="400px">
```
Comment (nicolasepiscopo) [inline 00-HTML-CSS-basics/html 1.1.4/html1.1.4.html]: You shouldn't set style properties for tables from html, do it from css with a class for your tables, rows and cells.

File: 00-HTML-CSS-basics/css 2.5/mock.html [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link rel="stylesheet" href="mock.css">
+    </head>
+    <body size="cover">
+        <div class="container" align="center">
+            <header id="header" class="myThings">Buenos Aires</header>
+            <br>
```
Comment (nicolasepiscopo) [inline 00-HTML-CSS-basics/css 2.5/mock.html:11]: we shouldn't use `<br>` as this is only for styling purpose, instead you can use a div element with a class, and from css you can set the height it should fill in order to create a blank space.. or better, add a margin bottom to your header

File: 00-HTML-CSS-basics/css 2.5/mock.html [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+<html class="no-js" lang="">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link rel="stylesheet" href="mock.css">
+    </head>
+    <body size="cover">
+        <div class="container" align="center">
+            <header id="header" class="myThings">Buenos Aires</header>
```
Comment (nicolasepiscopo) [inline 00-HTML-CSS-basics/css 2.5/mock.html]: For better readability use my-things instead of camelcased myThings when naming css classes

File: 00-HTML-CSS-basics/html 1.1.4/styles.css [hunk 11]
```diff
@@ -0,0 +1,27 @@
+#video {
```
Comment (nicolasepiscopo) [inline 00-HTML-CSS-basics/html 1.1.4/styles.css]: Instead of ids you should use classes for styling purposes, they are reusable!

File: 00-HTML-CSS-basics/html 1.1.5/html1.1.5.html [hunk 12]
```diff
@@ -0,0 +1,31 @@
+
```
Comment (nicolasepiscopo) [inline 00-HTML-CSS-basics/html 1.1.5/html1.1.5.html]: don't miss the doctype here

File: 00-HTML-CSS-basics/html 1.1.5/html1.1.5.html [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+	<head>
+		<title>Back to the future media</title>
+		<meta charset="utf-8">
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+		<link rel="stylesheet" href="styles.css">
+	</head>
+	<header><strong>Back to the future media</strong></header>
+	<body background="https://s-media-cache-ak0.pinimg.com/originals/e8/e5/38/e8e538fdacbc391abdf7f909315b7113.jpg">
```
Comment (nicolasepiscopo) [inline 00-HTML-CSS-basics/html 1.1.5/html1.1.5.html]: style only from css and not from html props

File: 00-HTML-CSS-basics/html 1.1.5/html1.1.5.html [hunk 14]
```diff
@@ -0,0 +1,31 @@
...
+		<DIV>
```
Comment (nicolasepiscopo) [inline 00-HTML-CSS-basics/html 1.1.5/html1.1.5.html]: write tag names in lowercase

File: 00-HTML-CSS-basics/html 1.1.6/html1.1.6.html [hunk 15]
```diff
@@ -0,0 +1,39 @@
...
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+		<link rel="stylesheet" href="styles.css">
+	</head>
+	<header><strong>Formulario</strong></header>
+	<body>
+		<div id="formulario">
+			<form id="formularioF" align="center">
+				Name: <br><input type="text" name="firstname"><br>
```
Comment (nicolasepiscopo) [inline 00-HTML-CSS-basics/html 1.1.6/html1.1.6.html]: Use the `<label for="inputName"></label>` tag

Comment (nicolasepiscopo) [reply 00-HTML-CSS-basics/html 1.1.6/html1.1.6.html]: http://www.w3schools.com/tags/tag_label.asp

File: 00-HTML-CSS-basics/html 1.1.6/html1.1.6.html [hunk 16]
```diff
@@ -0,0 +1,39 @@
...
+				Last name: <br><input type="text" name="lastname"><br>
+				Email: <br><input type="text" name="email"><br>
+				Birthday: <br><input type="text" name="birthday"><br>
```
Comment (nicolasepiscopo) [inline 00-HTML-CSS-basics/html 1.1.6/html1.1.6.html]: avoid the use of `<br>` tag

Comment (nicolasepiscopo) [reply 00-HTML-CSS-basics/html 1.1.6/html1.1.6.html]: use a css class for a `<label>` with `display: block` instead

File: 00-HTML-CSS-basics/html 1.1.6/html1.1.6.html [hunk 17]
```diff
@@ -0,0 +1,39 @@
...
+				Favorite team: <br><select name="favoriteTeam">
+					<option value="sl">San Lorenzo</option>
+					<option value="boca">Boca</option>
+					<option value="river">River</option>
+					<option value="racing">Racing</option>
+				</select> <br>
+				About yourself: <br><textarea name="miniBio" rows="5" cols="30"></textarea><br>
```
Comment (nicolasepiscopo) [inline 00-HTML-CSS-basics/html 1.1.6/html1.1.6.html]: avoid using rows and cols, use css width and min-height instead.

**Review by nicolasepiscopo — CHANGES_REQUESTED** [review]
